src/compiler/preprocess/index.ts

```diff
@@ -120,20 +122,27 @@ function processed_tag_to_code(
 	return tag_open_code.concat(content_code).concat(tag_close_code);
 }
 
+const regex_whitespace = /\s+/;
+const regex_unquoted_value = /^['"](.*)['"]$/;
```

Comment:
this name seems surprising to me, since it's looking for something quoted (but extracting something unquoted)

Reply:
I'll take a look at the implementation again and update the variable name